tubesize/src/styles/popup.css

```
@@ -95,17 +93,18 @@ body {
     background: rgba(255, 255, 255, 0.04);
     border-radius: 6px;
     cursor: pointer;
-    transition: all 0.2s;
     border: 1px solid transparent;
 }
 .format-item:hover {
     background: rgba(255, 255, 255, 0.08);
     border-color: rgba(255, 255, 255, 0.15);
+    border-color: #690b0b;
```

Biome (2.4.6)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)